app/test_split_integer.py

```diff
@@ -1,21 +1,23 @@
 from app.split_integer import split_integer
 
-
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
-
+    assert sum(split_integer(8, 3)) == 8
+    assert sum(split_integer(15, 4)) == 15
+    assert sum(split_integer(10, 5)) == 10
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
-
+    assert split_integer(8, 4) == [2, 2, 2, 2]
+    assert split_integer(12, 3) == [4, 4, 4]
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
-
+    assert split_integer(7, 1) == [7]
+    assert split_integer(15, 1) == [15]
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
-
+    result = split_integer(17, 4)
+    assert result == sorted(result)
+    assert max(result) - min(result) <= 1
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert split_integer(3, 5) == [0, 0, 0, 1, 2]
+    assert split_integer(2, 4) == [0, 0, 1, 1]
```

Comment:
The expected results in `test_should_add_zeros_when_value_is_less_than_number_of_parts` do not meet the task requirements. The task specifies that the array should be sorted in ascending order and the difference between the max and min number should be <= 1. Please adjust the expected results accordingly.